sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -24,40 +26,37 @@ public async Task RunSampleAsync()
             string moduleId = $"module{Random.Next(MaxRandomValue)}";
 
             // Create a DeviceIdentity.
-            DeviceIdentity deviceIdentity = await CreateDeviceIdentityAsync(deviceId);
+            await CreateDeviceIdentityAsync(deviceId);
 
             // Create a ModuleIdentity.
-            ModuleIdentity moduleIdentity = await CreateModuleIdentityAsync(deviceIdentity, moduleId);
+            await CreateModuleIdentityAsync(deviceId, moduleId);
 
             // List all modules within the device.
-            await ListAllModulesAsync();
+            await ListAllModulesAsync(deviceId);
 
             // Get the Module Identity.
-            await GetModuleIdentityAsync();
+            await GetModuleIdentityAsync(deviceId, moduleId);
 
             // Update Module Identity.
-            await UpdateModuleIdentityAsync();
+            await UpdateModuleIdentityAsync(deviceId, moduleId);
 
             // Get Module Twin,
-            await GetModuleTwinAsync();
+            await GetModuleTwinAsync(deviceId, moduleId);
```

Comment:
nit: ","

Reply:
I don't follow

Reply:
there is a `,` at the end of the comment, instead of a `.`!

Reply:
And on those lines, we should end all comments with a `.`.